pkgs/build-support/trivial-builders.nix

```diff
@@ -24,16 +24,16 @@ rec {
   * `allowSubstitutes = false;`
   * to a derivation’s attributes.
   */
-  runCommand = runCommandNoCC;
-  runCommandLocal = runCommandNoCCLocal;
+  runCommandNoCC = lib.warn "runCommandNoCC is deprecated. Use the equivalent runCommand function instead." runCommand;
```

Comment:
Do we plan removal at some point in the future? Now would be the time to announce that we will drop in in ~2022.11?

Reply:
I'd say it announces itself. No need to use up more brain bandwidth than necessary.

Reply:
Though, maybe link to the PR like we did for deprecating `stdenv.lib`. Not necessary though.